### ISSUE TITLE
App not working

### ISSUE THREAD

**Issue by diegogarciadev** [asuknath/Nagios-Status-JSON#30]
The application is not working, no notifications...

I'd like to improve the app, but the code need to be released to continue with development

**Comment by m-k-hasan** [comment]
Yes. I've uploaded "nath_status.php" file in my nagios server but in my Apps, the group key is not generated. What would be the solution?

**Comment by Webkungen** [comment]
The Group key is not generated in the app (Android). When clicking SHOW KEY nothing happens, and the app crashes. Still maintained?

**Comment by NH3CL** [comment]
> The Group key is not generated in the app (Android). When clicking SHOW KEY nothing happens, and the app crashes. Still maintained?
The last commit was 3 years ago, its abandoned sadly....

**Comment by NH3CL** [comment]
The last commit was 3 years ago, its abandoned sadly....
